app/javascript/packages/rails-i18n-webpack-plugin/spec/fixtures/expected946.js

```diff
@@ -1,22 +0,0 @@
-"use strict";
-(self["webpackChunkupaya"] = self["webpackChunkupaya"] || []).push([[946],{
```

Comment:
should this file have been `.gitignored`?

Reply:
I think originally I must have expected to use it as one of [the fixtures for testing the expected output of the Webpack plugin](https://github.com/18F/identity-idp/blob/f55ce3a778fa6250eabb4aef9ed9d432901ba755/app/javascript/packages/rails-i18n-webpack-plugin/rails-i18n-webpack-plugin.spec.js#L59-L76), but we're only comparing the contents of the language-suffixed output files, and that the base chunk is non-empty:

https://github.com/18F/identity-idp/blob/f55ce3a778fa6250eabb4aef9ed9d432901ba755/app/javascript/packages/rails-i18n-webpack-plugin/rails-i18n-webpack-plugin.spec.js#L73

It would have been simple enough to include the script in [the contents comparisons](https://github.com/18F/identity-idp/blob/f55ce3a778fa6250eabb4aef9ed9d432901ba755/app/javascript/packages/rails-i18n-webpack-plugin/rails-i18n-webpack-plugin.spec.js#L74-L76), but I realized I was probably intentional in not particularly caring about the actual contents of the chunk, so I figured it would be safe to remove altogether.